jobsdb/jobsdb.go

```diff
@@ -631,12 +648,21 @@ func (jd *HandleT) mapDSToLevel(ds dataSetT) (int, []int) {
 		jd.assertError(err)
 		return 1, []int{indexLevel0}
 	}
-	jd.assert(len(indexStr) == 2, fmt.Sprintf("len(indexStr): %d != 2", len(indexStr)))
+	if len(indexStr) == 2 {
```

Comment:
@soumyadebm @SumanthPuram @chandumlg This change is regarding ds numbering. Please review this.